### DISCUSSION TITLE
Msg: Uncaught (in promise) Error: "Browser" plugin is not implemented on android

### DISCUSSION THREAD

**Discussion by sharyuderoe** [quasarframework/quasar#15558] [CLI - Capacitor / Cordova modes]
I building mobile app using capacitor with quasar framework, but getting " Msg: Uncaught (in promise) Error: "Browser" plugin is not implemented on android" this error. can you please provide solution

**Comment by MilosPaunovic** [comment]
This is more `capacitor` related issue, so I would suggest you try to post it on their repo.
